styles.css

```diff
@@ -532,14 +532,25 @@ footer .footer-info h3 {
 footer form input,
 textarea {
   display: block;
-  font-family: "Poppins", sans-serif;
+  font-family: 'Poppins', sans-serif;
   width: 100%;
   margin-top: 18px;
   border: none;
   border-radius: 8px;
   padding: 15px 16px;
 }
 
+.email-submission-error {
+  margin-bottom: 20px;
+}
+
+.email-error {
+  color: red;
```

Comment:
- [ ] Let's change the color of the error. As it is right now it has a contrast of 1:1 with the background, which makes it really difficult to read. Let's try to improve this, which will result in a better user experience. I recommend you try to aim for 3:1+ contrast.

![image](https://user-images.githubusercontent.com/79114233/141323177-a8843da4-9713-4d0a-9319-4b9a27d016f1.png)